models/errors.go

```diff
@@ -32,15 +32,15 @@ func (err *Error) ToError() error {
 	return err
 }
 
-func (err *Error) Equal(other error) bool {
-	if e, ok := other.(*Error); ok {
-		if err == nil && e != nil {
-			return false
-		}
-		return e.GetType() == err.GetType()
-	}
-	return false
-}
+// func (err *Error) Equal(other error) bool {
+// 	if e, ok := other.(*Error); ok {
+// 		if err == nil && e != nil {
+// 			return false
+// 		}
+// 		return e.GetType() == err.GetType()
+// 	}
+// 	return false
+// }
```

Comment:
Delete or uncomment depending on if it's needed still?

Reply:
Cleaned up in [f7314a9](https://github.com/cloudfoundry/bbs/pull/109/commits/f7314a95de15266d952902af8767dd6b5a741df3)